cypress/component/deletable-listitem.cy.tsx

```diff
@@ -34,25 +34,12 @@ describe('DeletableListItem', () => {
       ],
     };
 
-    const softUndo = () => {
-      cy.stub();
-    };
-
-    const softUndoSpy = cy.spy(softUndo).as('softUndoSpy');
-
-    const softDelete = () => {
-      cy.stub();
-    };
-
-    const softDeleteSpy = cy.spy(softDelete).as('softDeleteSpy');
-
     cy.mount(
       <List>
         <DeletableListItem
-          overviewOrg={overviewOrg}
-          softDeleteCallback={softDeleteSpy}
-          softUndoCallback={softUndoSpy}
-          item={overviewOrg.listItems[0]}
+          softDeleteCallback={() => null}
+          softUndeleteCallback={() => null}
```

Comment:
Yeah me too, but I think softUndo is a bad word too because I wanted it to be more clear that this function did the opposite of softDelete. Open for suggestions for the name though.

Reply:
I like softRestore